unix.cpp

```diff
@@ -1,16 +1,60 @@
 #include "system.h"
 #include "settings.h"
 #include <QDir>
+#include <QStandardPaths>
+#include <QString>
 
 namespace Sys {
 QString archiveName(void)
 {
     return "linux64.zip";
 }
 
+void migrateHomePath(void)
```

Comment:
Was the updater ever used by anyone in "production"? Because if not, we don't need to have a function to migrate data (and thus there should not be one because less code is better code).

Reply:
The only good reason I see for data migration is to allow users to change their install location at any point in time after they started the initial install (even *during* initial install or during an update) – so you might want to keep the code but turn `legacyHomePath` into a pair of paramters named `oldInstallPath` and `newInstallPath` or something.

Reply:
It has never been widely used. I think this is valid for the 5-10 people who may have used it...

Reply:
Note that a shell equivalent is:

> basepath="unvanquished"; legacypath="${HOME}/.${basepath}"; xdgpath="${XDG_DATA_HOME:-${HOME}/.local/share}/${basepath}"; if [ -d "${legacypath}" -a ! -d "${xdgpath}" ]; then if [ -h "${legacypath}" ]; then ln -sv "${legacypath}" "${xdgpath}"; else mv -v "${legacypath}" "${xdgpath}"; fi; fi